src/components/ReviewById.jsx

```diff
@@ -0,0 +1,14 @@
+// import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import ReviewIdCard from "./ReviewIdCard";
+// import { getReviewById } from "../utils.js";
+
+export default function ReviewById() {
+  const { reviewId } = useParams();
+
+  return (
+    <section>
+      <ReviewIdCard reviewId={reviewId} />
+    </section>
+  );
```

Comment:
Is this component necessary? Could this be rendered somewhere else? It feels as though this is slightly over engineered and there are more components going on that necessary 💭

Reply:
Thanks Kirsty, this has now been refactored.